dm/master/server.go

```diff
@@ -458,7 +462,9 @@ func (s *Server) StartTask(ctx context.Context, req *pb.StartTaskRequest) (*pb.S
 				s.removeMetaLock.Unlock()
 				return resp, nil
 			}
-			err = s.removeMetaData(ctx, cfg)
+			toDB := cfg.TargetDB
+			toDB.Adjust()
```

Comment:
`Adjust` can also be moved into `removeMetaData`. And please check we should clone the toDB when `Adjust`, to avoid unexpected changes.

Reply:
fixed in https://github.com/pingcap/dm/pull/1999/commits/a98c9a72795f4c6e7cc7f5fee1ad7a7a27c0d229
there is no need to clone a configuration yet, as `cfg.TargetDB` is not used again in the following logic